packages/communicator-email-ses/index.ts

```diff
@@ -0,0 +1,152 @@
+import {Communicator, CommunicatorContext, Configurable, InitContext} from '@dbos-inc/dbos-sdk';
+
+import { SESv2, SendEmailCommand } from '@aws-sdk/client-sesv2';
+import { AWSServiceConfig, getAWSConfigForService, getAWSConfigs } from '@dbos-inc/aws-config';
+
+interface SESConfig{
+    awscfgname?: string,
+    awscfg?: AWSServiceConfig,
+}
+
+@Configurable()
+class SendEmailCommunicator
+{
+    static AWS_SES_CONFIGURATIONS = 'aws_ses_configurations';
+    static async initConfiguration(ctx: InitContext, arg: SESConfig) {
+        // Get the config and call the validation
+        getAWSConfigs(ctx, SendEmailCommunicator.AWS_SES_CONFIGURATIONS);
```

Comment:
Why are we calling getAWSConfigs but then not looking at the return value?

Reply:
It throws an exception when there isn't a config or it is malformed.